CONTRIBUTING.rst

```diff
@@ -7,7 +7,7 @@ new submissions, or contributing code that can be incorporated into the
 project.
 
 Please, don't use the issue tracker for support questions. Instead use:
```

Comment:
We have a `question` issue type, maybe we have to change this line?

Reply:
fixed